### ISSUE TITLE
Push the CoreDNS 1.2.5 image in gcr.io

### ISSUE THREAD

**Issue by rajansandeep** [kubernetes/kubernetes#70425]
Need to push the latest CoreDNS image to gcr.io
The 1.2.5 image is available in docker coredns/coredns:1.2.5

 
This is the script used for pushing the image:

```
#!/bin/sh

if [ -z "$1" -o -z "$2" -o -z "$3" -o -z "$4" ]; then
    echo "usage: $0 <src-reg> <image> <tag> <dst-reg>" > /dev/stderr
    echo "example: $0 coredns coredns 1.2.5 staging-k8s.gcr.io" > /dev/stderr
    exit 1
fi

SRC_REG="$1"
IMG="$2"
TAG="$3"
DST_REG="$4"
TMP="/tmp/$RANDOM"
JSON="$TMP.json"

echo "Fetching source manifest"
docker manifest inspect ${SRC_REG}/${IMG}:${TAG} > ${JSON}

echo
echo "Pulling source images by SHA"
cat ${JSON} \
    | jq -j ".manifests[] | \"${SRC_REG}/${IMG}@\", .digest, \"\n\" " \
    | while read X; do docker pull $X; done

echo
echo "Tagging images with ARCH and OS"
cat ${JSON} \
    | jq -j ".manifests[] | \"docker tag ${SRC_REG}/${IMG}@\", .digest, \" ${DST_REG}/${IMG}:${TAG}__\", .platform.architecture, \"_\", .platform.os, \"\n\"" \
    | while read X; do $X; done

echo
echo "Pushing images"
cat ${JSON} \
    | jq -j ".manifests[] | \"gcloud docker -- push ${DST_REG}/${IMG}:${TAG}__\", .platform.architecture, \"_\", .platform.os, \"\n\"" \
    | while read X; do $X; done

echo
echo "Creating and pushing manifest"
PLATFORMS=$(cat ${JSON} | jq -j ".manifests[] | .platform.os, \"/\", .platform.architecture, \"\n\"" | paste -sd,)
echo manifest-tool push from-args \
    --platforms "${PLATFORMS}" \
    --template ${DST_REG}/${IMG}:${TAG}__ARCH_OS \
    --target ${DST_REG}/${IMG}:${TAG}
manifest-tool push from-args \
    --platforms "${PLATFORMS}" \
    --template ${DST_REG}/${IMG}:${TAG}__ARCH_OS \
    --target ${DST_REG}/${IMG}:${TAG}
```

/cc @chrisohaver 
/cc @BenTheElder
/cc @timothysc
/cc @AishSundar 
/cc @dims 
/assign @thockin

@fturib 

/sig network
/sig cluster-lifecycle
/sig release

**Comment by thockin** [comment]
Why closing?  I am having an issue with manifest-tool and auth'ed repos, but once I get past that - tell me whats up?

**Comment by chrisohaver** [comment]
@thockin, a bug in forwarding loop detection in 1.2.5 was fixed, we plan to push 1.2.6 with the fix instead. coredns/coredns#2255
